lib/Screens/Auth/signup_screen.dart

```diff
@@ -0,0 +1,210 @@
+}
+
+class _SignUpScreenState extends State<SignUpScreen> {
+  final emailController = TextEditingController();
+  final passwordController = TextEditingController();
+  final nameController = TextEditingController();
+  final cpasswordController = TextEditingController();
+  @override
+  void dispose() {
+    super.dispose();
+    emailController.dispose();
+    passwordController.dispose();
+    nameController.dispose();
+    cpasswordController.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return SafeArea(
+      child: Scaffold(
+          body: SingleChildScrollView(
+        child: Stack(
+          fit: StackFit.passthrough,
+          children: [
+            Container(
+              width: 510,
+              height: 790,
+              color: Colors.white,
+            ),
+            Container(
+                height: 320,
+                width: 410,
+                decoration: const BoxDecoration(
+                  image: DecorationImage(
+                    image: AssetImage('assets/images/bg.png'),
+                    fit: BoxFit.cover,
+                  ),
+                ),
+                child: const Padding(
+                  padding: EdgeInsets.only(right: 300, bottom: 220),
+                  child: Image(image: AssetImage('assets/images/logo.png')),
+                )),
+            Positioned(
+              bottom: 0,
+              left: 0,
+              child: SizedBox(
+                height: 570,
+                width: 410,
+                child: Card(
+                    surfaceTintColor: Colors.white,
+                    shape: const RoundedRectangleBorder(
+                        borderRadius: BorderRadius.only(
+                            topLeft: Radius.circular(45),
+                            topRight: Radius.circular(45))),
+                    child: SizedBox(
+                      width: double.infinity,
+                      height: 500,
+                      child: Column(
+                        children: [
+                          const Padding(
+                            padding: EdgeInsets.only(top: 40),
+                            child: Text(
+                              'CREATE ACCOUNT',
+                              style: TextStyle(
+                                fontSize: 30,
+                                fontWeight: FontWeight.w500,
+                              ),
+                            ),
+                          ),
+                          const SizedBox(
+                            height: 30,
+                          ),
+                          Column(children: [
+                            SizedBox(
+                              width: 320,
+                              height: 52,
+                              child: TextField(
+                                controller: nameController,
+                                decoration: InputDecoration(
```

Comment:
The current `highlightColor` of the `TextField` is light bluish, would be better if you can change it to a slightly darker one